util/src/main/java/org/hyperledger/besu/util/NetworkUtility.java

```diff
@@ -173,4 +196,20 @@ private static boolean isPortAvailableForUdp(final int port) {
   public static boolean isPortAvailable(final int port) {
```

Comment:
I've had a look at the isPortAvailableFor[Tcp|Udp] and I thought that we might want to close the ServerSocket if we successfully binded to it?

Reply:
Apparently SO_REUSEADDR makes it moot, but I don't see why we wouldn't unbind.  It isn't in the scope of this PR, but I can put up a subsequent one.